frontend/src/App.vue

```diff
@@ -118,25 +155,35 @@
 </template>
 
 <script lang="ts">
-import Vue from "vue";
 import Component from "vue-class-component";
-import { Watch } from "vue-property-decorator";
+import { Mixins, Watch } from "vue-property-decorator";
 import TheSnackbar from "@/components/TheSnackbar.vue";
 import { RouteTag } from "./router";
+import UserInfoApiRequestMixin from "@/mixins/requests/UserInfoApiRequestMixin";
+import { Userinfo } from "./types/common/Userinfo";
 
 @Component({
-  components: { TheSnackbar }
+  components: { TheSnackbar },
 })
-export default class App extends Vue {
-
+export default class App extends Mixins(UserInfoApiRequestMixin) {
   public query = "";
+  private userinfo = new Userinfo();
+
+  private fav = true;
+  private menu = false;
```

Comment:
Den Datentyp für die Variable `menu` definieren. In Klassen und Komponenten zwischen den Variablen immer eine Zeile abstand.